proto/org/hypertrace/agent/config/v1/config.proto

```diff
@@ -1,11 +1,11 @@
 syntax = "proto3";
 
-package org.hypertrace.agent.config;
+package org.hypertrace.agent.config.v1;
 
 import "google/protobuf/wrappers.proto";
 
-option go_package = "github.com/hypertrace/goagent/config";
-option java_package = "org.hypertrace.agent.config";
+option go_package = "github.com/hypertrace/goagent/config/v1";
```

Comment:
@jcchavezs now we're checking these in, and TA is going to stop importing these protos directly, can we remove the goagent?  And instead use:
`github.com/hypertrace/agent-config/gen/go/hypertrace/agent/config/v1` ?

Reply:
If we check in the generated code we can just get rid of the submodule and generation in goagent.